aws/resource_aws_rds_cluster.go

```diff
@@ -96,9 +96,18 @@ func resourceAwsRDSCluster() *schema.Resource {
 			},
 
 			"engine": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "aurora",
+				ForceNew:     true,
+				ValidateFunc: validateRdsEngine,
```

Comment:
I don't think we should validate the engine value, as we would need to update the code each time a new engine is available. Thoughts?

Reply:
Valid point however, amazon introduced aurora for MySQL in 2014 and added PostgreSQL in 2017. Don't have insights into their product plans but so far if they keep adding one every 3 years updating code won't be an issue. I think having validation is more useful/user friendly than flexibility.

Reply:
Looks good to me :)